pkg/enabled_linters.go

```diff
@@ -125,6 +126,7 @@ func GetAllSupportedLinterConfigs() []LinterConfig {
 		newLinterConfig(golinters.Maligned{}).WithFullImport().WithPresets(PresetPerformance).WithSpeed(10),
 		newLinterConfig(golinters.Megacheck{GosimpleEnabled: true, UnusedEnabled: true, StaticcheckEnabled: true}).
 			WithSSA().WithPresets(PresetStyle, PresetBugs, PresetUnused).WithSpeed(1),
+		newLinterConfig(golinters.Depguard{}).WithPresets(PresetAdmin).WithSpeed(6),
```

Comment:
why not add it to `style` preset?

Reply:
`WithFullImport()` is needed: without that it won't load `loader.Program` if only this linter is enabled

Reply:
I didn't really think it fit the style prefix but I will gladly move it if you think that is more proper. I was thinking it from a repository maintainer administrating what imports are allowed. Let me know what you think?

Reply:
I think it's a kind of style guide: what allowed and what not.